clients/runtime/src/rpc.rs

```diff
@@ -525,6 +525,7 @@ pub trait VaultRegistryPallet {
 	async fn get_required_collateral_for_wrapped(
 		&self,
 		amount_wrapped_asset: u128,
+		wrapped_currency_id: CurrencyId,
 		collateral_currency: CurrencyId,
```

Comment:
Nit: I'd rename `collateral_currency` to `collateral_currency_id` to make it consistent.

Reply:
Done